clamav/client.py

```diff
@@ -61,3 +66,37 @@ def health(self):
         url = self.routes.health()
         response = self._request(self._session.get, url)
         return ClamAVHealth(response.json())
+
+    def scan_container_image(self, image_reference: str) -> typing.Tuple[ClamAVScanResult, str]:
```

Comment:
nit/tip:
```
FilePath = str
...
typing.Tuple[ClamAVScanResult, FilePath]
```
would make the signature more expressive.

as an alternative, why not augment the ClamAvScanResult obj with the path? We do not need to restrict our model classes to be 100% identical to ClamAV's raw data